examples/jsm/tsl/display/CANode.js

```diff
@@ -0,0 +1,310 @@
+import { Vector2, TempNode } from 'three/webgpu';
+import {
+	nodeObject,
+	Fn,
+	uniform,
+	convertToTexture,
+	float,
+	vec4,
+	uv,
+	NodeUpdateType,
+} from 'three/tsl';
+
+/**
+ * Post processing node for applying chromatic aberration effect.
+ * This effect simulates the color fringing that occurs in real camera lenses
+ * by separating and offsetting the red, green, and blue channels.
+ *
+ * @augments TempNode
+ * @three_import import { chromaticAberration } from 'three/addons/tsl/display/ChromaticAberrationNode.js';
+ */
+class ChromaticAberrationNode extends TempNode {
```

Comment:
How about renaming the file `CANode` to the class name `ChromaticAberrationNode`? I know it's a long name, but we usually follow this logic.

Reply:
Or we rename the class to `CANode` (similar to `GTAONode`, `SMAANode` or `SSRNode`). No matter how we decide, module and class name should be in sync.

Reply:
I agree, I would just vote for `ChromaticAberrationNode` as it seems like a more common name.

Reply:
I'm okay with that!